test/integration/controllers/execute-command.test.js

```diff
@@ -0,0 +1,74 @@
+const supertest = require('supertest');
+const { expect } = require('chai');
+const sinon = require('sinon');
+
+describe('API Execute command', function () {
+  let clock;
+  beforeEach(() => {
+    clock = sinon.useFakeTimers();
```

Comment:
Wazzthis do?

Reply:
'freezes time'. Part of the API response is the current date, which didn't work very well for assertions

Reply:
ah, so unlike mockdate that set the time, this prevents it from moving forward
gotcha, good tool to know

Reply:
Pretty much, it's also useful when dealing with `setTimeout` for example, see the test for the tele command